src/tokens.scss

```diff
@@ -0,0 +1,6 @@
+:host {
+  --ds-auro-menu-border-color: var(--ds-color-border-divider-default, $ds-color-border-divider-default);
```

Comment:
What do you think about changing this name? The moment I read it I thought you were styling a border around the whole menu. 

This is only used to create a divider look between grouped menus.

Maybe `--ds-auro-menu-divider-color`?